ci/conda/recipes/morpheus-libs/meta.yaml

```diff
@@ -147,9 +145,18 @@ outputs:
 
     requirements:
       build:
+        - {{ compiler("c") }}
+        - {{ compiler("cuda") }}
+        - {{ compiler("cxx") }}
         - ccache
         - cmake =3.27
+        - ninja =1.11
+        - pkg-config =0.29
       host:
+        # cuda version is pinned here. without this the resolver pulls
+        # in versions that are not compatible with morpheus-core
+        - cuda-cudart-dev {{ cuda_compiler_version }}.*
```

Comment:
Is this turning a build requirement into a runtime requirement?

Reply:
It is not. The build vs host requirements in the meta.yaml are a bit confusing - [conda guide](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements)

Both the sections are for building the package. Host requirements are to fetch the dependencies by host platform. In our case both build and host platforms are x86_64 (at the moment) however the resolver is behaving differently if they (the cuda dev libs) are NOT listed in the host section; eventually failing to set up the build env (because of version mistmatches). That is the reason for including them in the host section of all the outputs (unfortunately there is no concept of inheritance across output sections).

I dropped the duplicates from the build as it seemed unnecessary/confusing.